charts/docker-hub-rate-limit-exporter-chart/ci/all-enabled-values.yaml

```diff
@@ -2,22 +2,45 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-config:
-    verbosity: 1 ## the loglevel
-    dockerhub:
-        username: "" ## put your dockerhub username here
-        password: "" ## put your dockerhub password here
-
-deployment:
-    enabled: false
-    replicaCount: 1
-
+# Commonly used for authenticated users
+deployments:
+    default:
```

Comment:
I like the idea of having the possibility to track multiple dockerhub accounts. However, I think this problem is a little bit more complex than the suggested implementation here. E.g. I think we would also need multiple ServiceMonitor Resources - one for each deployment. Also the existing Grafana Dashboard can not handle multiple deployments yet either. There is an discussion regarding the different possibilities of implementing a multi account monitoring here: https://github.com/viadee/docker-hub-rate-limit-exporter/issues/5

Moreover, I implemented a small proof of concept a while ago, you can find it here: 
https://github.com/viadee/docker-hub-rate-limit-exporter/tree/poc/multiple-dockerhub-accounts

However, I don't think we should implement that feature with this pull request. The changes already included in this PR represent a big improvement on its own and implementing the multi-account feature would definitely require some more work. Do you mind reverting the deployment structure here? 

Of course, if you also want to help with the mulit-account feature, you are more than welcome to open another Pull request for that feature. :-)

Reply:
Okay so i'm gonna put it back like before and open another MR for this multi account thing